concepts/switch/about.md

```diff
@@ -0,0 +1,60 @@
+# About
+
+Like other languages, C++ also provides a `switch` statement.
+Switch statements are a shorter way to write long `if ... else if` statements.
+To make a switch, we start by using the keyword `switch` followed by an integer.
+We then declare each one of the conditions with the `case` keyword.
+We can also declare a `default` case, that will run when none of the previous `case` conditions matched.
+Each case should end with a `break` (or a `return`) statement.
+
+```cpp
+int price{0};
+int adults{3};
+int kids{2};
+
+switch (int group_size{adults + kids}) {
+    case 1:
+        price = 50;
+        break;
+    case 2:
+        price = 70;
+        break;
+    default:
+        price = group_size * 30;
+}
+```
+
+## Fall-through
+
+One important thing about the switch construct is that the code will continue to be executed until it stops at a `break` statement.
+This can lead to unexpected behavior.
```

Comment:
Coming from Python, I wanted more explanation here as to why this behavior happens -- and why you would or would not want it to happen.

I also confuse myself often with `fall-through` vs `default`, because it is hard for me to remember that code execution continues past a match unless explicitly ended by `break;` or `return`.